spec/factories/sponsors.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :sponsor do
+    event
+    name { "Fake Sponsor" }
```

Comment:
It might be more consistent with the rest of the test suite to use the Faker gem to generate names for the sponsors.

Reply:
Got it.